odl/solvers/advanced/proximal_operators.py

```diff
@@ -446,6 +436,33 @@ def proximal_box_constraint(space, lower=None, upper=None):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    If :math:`P` is an interval :math:`[a,b]`, the indicator function is
+    defined as
+
+        .. math::
+
+            I_{x \\in P} = \\left\{ \\begin{array}{ll}
+            0 & \\quad \\text{if } x \\in P, \\\\
+            \\infty & \\quad \\text{if } x \\not \\in P
+            \\end{array} \\right.
+
+    For a step size :math:`\\sigma`, the proximal operator of
+    :math:`\\sigma I_{x \\in P}` is given by the projection onto the interval
+
+      .. math::
+
+             prox[\\sigma I_{x \\in P}](x) = \\left\{ \\begin{array}{ll}
```

Comment:
I don't quite like our old style of `prox[functional](point)`, i.e. the functional in square parentheses. IMO it looks better as subscript and upright font, i.e. `\mathrm{prox}_{\\iota_P}(x)`. Should be fixable by simple search-and-replace.